plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -2262,6 +2263,7 @@ private void removeOrphanFiles(Table table, ConnectorSession session, SchemaTabl
                     continue;
                 }
 
+                verifyManifestFileExists(fileSystem, manifest.path());
```

Comment:
I feel just adding another `catch(NotFoundException)` to the following try-catch  is sufficient. Any reason we should explicitly check the existence?

Reply:
ah, you are right!